mainjsdoshop

```diff
@@ -2,18 +2,43 @@
 
 
 var DonutShop = function(locationName, avgMin, avgMax, avgPerCustomer) {
-  this.locationName = locationName
+  this.locationName = locationName;
   this.avgMin = avgMin;
   this.avgMax = avgMax;
-  this.avgPerCustoemr = avgPerCustomer;
-  this.myArray = [];
-  this.total = 0;
-
+  this.avgPerCustomer = avgPerCustomer;
 
 DonutShop.prototype.nutPerHour = function() {
-this.total = this.myArray
-this.myArray = Math.floor(Math.random() * (this.avgMax-this.avgMin + 1) + (this.avgMin * this.avgPerCustomer));
-}
+  this.totalSales = 0;
+  this.hourlyNutArray = [];
+  for(var i = 0; i < 12; i++) {
+  this.hourlyNutArray.push(Math.floor((Math.random() * (this.avgMax-this.avgMin + 1) + this.avgMin) * this.avgPerCustomer));
+  this.totalSales = this.totalSales + this.hourlyNutArray[i];
```

Comment:
It's more common to use a `+=` operator, which will save you some typing. It would look like this:  `this.totalSales += this.hourlyNutArray[i]`